src/control/cmd/ddb/ddb_commands.go

```diff
@@ -81,7 +82,16 @@ pool shard. Part of the path is used to determine what the pool uuid is.`,
 		LongHelp:  "",
 		HelpGroup: "vos",
 		Run: func(c *grumble.Context) error {
-			return ddbSuperblockDump(ctx)
+			sb, err := ddbSuperblockDump(ctx.CContext)
+			if err != nil {
+				return err
+			}
+
+			log := logging.NewCommandLineLogger()
+			err = printSuperBlock(log, sb, ctx.jsonOutput)
+			ctx.jsonOutputHandled = true
```

Comment:
Instead of overloading `printSuperBlock()` to handle both types of output, it would be better to just use [cmdutil.OutputJSON()](https://github.com/daos-stack/daos/blob/master/src/control/common/cmdutil/json.go#L31) to emit JSON directly to `os.Stdout`. That will simplify your code and also not rely on the implementation of the default command line logger which currently emits unadorned output at INFO level, but that could change.

Reply:
Looks like cmdutil.OutputJSON was just added to master. Will need to wait for master to be merged into feature/cat_recovery to take advantage, but I will make this change.

Reply:
Master was merged into feature/cat_recovery so updated to use cmdutil.OutputJSON